package.json

```diff
@@ -15,7 +15,7 @@
   },
   "devDependencies": {
     "@ionic/prettier-config": "^2.0.0",
-    "@stencil/core": "^2.9.0",
```

Comment:
I don't _think_ NPM knows how to bump packages with text in their version string. e.g. I'm not sure if it knows that `v3.0.0-rc.0` (the next type of release we'll do) is a higher version than `v3.0.0-beta.0`. But maybe I'm wrong there - do you know if that's the case?

Ideally, I don't want to have to put up a PR when we go from beta->rc->ga, mostly so we don't have as tight of a coordination schedule when we're rolling v3.0.0 out. In the event that NPM doesn't know that rc comes after beta (and that `3.0.0` comes after rc), I'm not sure what the best course of action is here. IIRC npm can get finicky here. Maybe using `v3-next` in our accepted versions string in addition to `^3.0.0`? I'm not 100% sure npm will accept that without doing something like:
1. Installing v3-next with `npm i @stencil/core@v3-next`
2. Manually edit the dependencies
3. Installing the latest Stencil Core (on v2) with `npm i @stencil/core@latest`)

Reply:
Yeah I'm not 100% sure on some of these range matching bits, but I went ahead with this range after checking here: https://jubianchi.github.io/semver-check/#/^3.0.0-beta.0/3.0.0

and doing some reading of the `semver` package documentation, here: https://github.com/npm/node-semver

I _believe_ that, for instance, `3.0.0` or `3.0.1` will be accepted as meeting `^3.0.0-beta.0` (as will, _I think_, `3.0.0-beta.1`, `3.0.0-rc.0`, etc).